app.js

```diff
@@ -2,26 +2,52 @@
 
 const express = require('express');
 const app = express();
+const gapi = require('./domains');
```

Comment:
Oof, I think we have done a poor job communicating here. I wasn’t thinking it would be deleted at all. :\